### PR TITLE
Update install-cockroachdb.html

### PR DESCRIPTION
**Pull request by knz**
Reported by @richardwu

### CODE REVIEW

File: v1.1/install-cockroachdb.html [hunk 1]
```diff
@@ -374,6 +374,13 @@ <h2 id="whats-next">What's Next?</h2>
         <td>Autoconf</td>
         <td>Version 2.68 or higher is required.</td>
       </tr>
+      <tr>
+        <td>Terminfo / NCurses development libraries</td>
+        <td>This may be
+            part of a ncurses development package (e.g. `libtinfo-dev` on
```
Comment (richardwu) [inline v1.1/install-cockroachdb.html]: I think you'll need to wrap `libtinfo-dev` in `<code>libtinfo-dev</code>` (also `ncurses-devel`).

File: v1.1/install-cockroachdb.html [hunk 2]
```diff
@@ -374,6 +374,13 @@ <h2 id="whats-next">What's Next?</h2>
         <td>Autoconf</td>
         <td>Version 2.68 or higher is required.</td>
       </tr>
+      <tr>
+        <td>Terminfo / NCurses development libraries</td>
```
Comment (richardwu) [inline v1.1/install-cockroachdb.html]: This doesn't wrap too nicely, maybe just `Terminfo/NCurses`?

Comment (knz) [reply v1.1/install-cockroachdb.html]: That would be inaccurate. Folk usually already have libncurses installed, but it's not sufficient.

File: v1.1/install-cockroachdb.html (hunk 1 above)
Comment (knz) [reply v1.1/install-cockroachdb.html]: Done.

**Comment by jseldess** [comment]
Thanks, @knz. Mind if I take this over and apply the update to the 1.2 docs as well?

**Comment by knz** [comment]
Please do

**Comment by knz** [comment]
Oh wait, actually I made a mistake: this change should **only** be applied to the 1.2 docs. It was an error to change this for 1.1 -- v1.1 is not affected.

**Comment by jseldess** [comment]
OK, @knz. I'll move the change just to the 1.2 version of the page.

**Pull request merged**